src/Refactoring-UI-Tests/ReExtractToTemporaryDriverTest.class.st

```diff
@@ -49,35 +50,53 @@ ReExtractToTemporaryDriverTest >> testExtractToTemporaryExpectChanges [
 { #category : 'tests' }
 ReExtractToTemporaryDriverTest >> testFailureInvalidVariableName [
 
-	| driver rbclasses |
+	| driver rbclasses dialog requestClass|
 	rbclasses := RBClassEnvironment classes: { RBClassDataForRefactoringTest }.
 	driver := ReExtractToTemporaryDriver new
 		scopes: { rbclasses };
 		extract: (51 to: 60)
-      to: '1collection'
       from: #modifiesCollection
       in: RBClassDataForRefactoringTest.
 
    self setUpDriver: driver.
 
-	self setUpMocksOn: driver.
+    dialog := MockObject new.
+    dialog 
+        on: #variableName
+        respond: (RBArgumentName name: 'collection').
+    driver requestDialog: dialog.
+    requestClass := MockObject new.
+    requestClass
+        on: #openOn:
+        respond: dialog.
+    driver variableNameEditorPresenterClass: requestClass.
 
 	self should: [ driver runRefactoring ] raise: RBRefactoringError.
 ]
```

Comment:
Can you include some information in the test method name on why is this invalid variable name?